config/initializers/trestle.rb

```diff
@@ -0,0 +1,134 @@
+  # Rails application's routes. Defaults to true.
+  #
+  # config.automount = false
+
+  # == Navigation Options
+  #
+  # Set the path to consider the application root (for title links and breadcrumbs).
+  # Defaults to the same value as `config.path`.
+  #
+  # config.root = "/"
+
+  # Set the initial breadcrumbs to display in the breadcrumb trail.
+  # Defaults to a breadcrumb labeled 'Home' linking to to the application root.
+  #
+  # config.root_breadcrumbs = -> { [Trestle::Breadcrumb.new("Home", Trestle.config.root)] }
+
+  # Set the default icon class to use when it is not explicitly provided.
+  # Defaults to "fa fa-arrow-circle-o-right".
+  #
+  # config.default_navigation_icon = "fa fa-arrow-circle-o-right"
+
+  # Add an explicit menu block to be added to the admin navigation.
+  #
+  # config.menu do
+  #   group "Custom Group" do
+  #     item "Custom Link", "/admin/custom", icon: "fa fa-car", badge: { text: "NEW!", class: "label-success" }, priority: :first
+  #   end
+  # end
+
+  # == Extension Options
+  #
+  # Specify helper modules to expose to the admin.
+  #
+  # config.helper :all
+
+  # Register callbacks to run before, after or around all Trestle actions.
+  #
+  # config.before_action do |controller|
+  #   Rails.logger.debug("Before action")
+  # end
+  #
+  # config.after_action do |controller|
+  #   Rails.logger.debug("After action")
+  # end
+  #
+  # config.around_action do |controller, block|
+  #   Rails.logger.debug("Around action (before)")
+  #   block.call
+  #   Rails.logger.debug("Around action (after)")
+  # end
+
+  # Specify a custom hook to be injected into the admin.
+  #
+  # config.hook(:stylesheets) do
+  #   stylesheet_link_tag "custom"
+  # end
+
+  # Toggle whether Turbolinks is enabled within the admin.
+  # Defaults to true if Turbolinks is available.
+  #
+  # config.turbolinks = false
+
+  # Specify the parameters that should persist across requests when
+  # paginating or reordering. Defaults to [:sort, :order, :scope].
+  #
+  # config.persistent_params << :query
+
+  # Customize the default adapter class used by all admin resources.
+  # See the documentation on Trestle::Adapters::Adapter for details on
+  # the adapter methods that can be customized.
+  #
+  # config.default_adapter = Trestle::Adapters.compose(Trestle::Adapters::SequelAdapter)
+  # config.default_adapter.include MyAdapterExtensions
+
+  # Register a form field type to be made available to the Trestle form builder.
+  # Field types should conform to the following method definition:
+  #
+  # class CustomFormField
+  #   def initialize(builder, template, name, options={}, &block); end
+  #   def render; end
+  # end
+  #
+  # config.form_field :custom, CustomFormField
+
+  # == Debugging Options
+  #
+  # Enable debugging of form errors. Defaults to true in development mode.
+  #
+  # config.debug_form_errors = true
+
+
+  config.hook("view.header") do
+    render "head"
```

Comment:
I see you had to put a this head file in each admin resource. Try changing this to `render "admin/head"` and then you may not need to create multiple head files